Dockerfile

```diff
@@ -12,10 +12,14 @@ USER ${NB_UID}
 
 # Install Tensorflow
 RUN pip install --quiet --no-cache-dir \
+    uproot \
```

Comment:
make Dockerfile the same as mine: https://github.com/jmduarte/L1METML/blob/master/Dockerfile
(i.e. remove `keras`)